app/views/vacancies/_job_details_section.html.haml

```diff
@@ -57,13 +58,13 @@
     %h2.govuk-heading-m= t('jobs.leadership_level')
     %p= @vacancy.leadership.title
 
-  - if @vacancy.show_supporting_documents? && !@vacancy.documents.none?
+  - if @vacancy.show_supporting_documents? && @vacancy.documents.any?
     %h2.govuk-heading-m= t('jobs.supporting_documents')
     .supporting-docs
       %ul
         - @vacancy.documents.each do |document|
           %li
-            %a.document-link{ href: document[:download_url], target: '_blank' }
+            .document-link{ href: document[:download_url], target: '_blank' }
```

Comment:
I'm not sure this will do what you're thinking, but let's try it.

Reply:
I forgot to remove the attributes when fixing merge conflicts, my bad